rhino/src/main/java/org/mozilla/javascript/CompilerEnvirons.java

```diff
@@ -98,13 +98,23 @@ public void setXmlAvailable(boolean flag) {
         xmlAvailable = flag;
     }
 
+    @Deprecated
```

Comment:
I really like the patter we are using in HtmlUnit; this is a great help if you later on have to decide to remove this stuff

     *
     * @deprecated as of version 4.5.0; use {@link #validateSelectors(SelectorList, DomNode)} instead
     */
    @Deprecated

Reply:
Yes, I will go in and add this syntax, thanks!